modules/planning/common/planning_context.h

```diff
@@ -40,6 +40,8 @@
 namespace apollo {
 namespace planning {
 
+constexpr int kPathScenarioTransitionHysteresisFrame = 5;
```

Comment:
Local vars should be either `static` or in anonymous namespace to avoid global namespace pollution.

Reply:
I will put all such thing into GFLAGS at the very end when dev is finished. They are meant to be used globally.